ospfd/ospf_lsa.c

```diff
@@ -1043,6 +1057,9 @@ void ospf_network_lsa_update(struct ospf_interface *oi)
 	if (new == NULL)
 		return;
 
+	if (!new)
```

Comment:
Isn't this redundant with the three lines above it?

Reply:
Yeah, I dunno, I just thought it would be good to explicitly check the return value as well.

Reply:
but that is exactly what line 1057 is there for, right?

Reply:
oh what the hell, how did those get there